benchmarks/transformers/routing.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+
+class RouteCQC:
+    params = [[10, 25, 50, 75, 100], [10, 50, 100, 250, 500, 1000], [0.5], [10]]
+    param_names = ["qubits", "depth", "op_density", "grid_device_size"]
+    timeout = 120  # Increase timeout to 2 minutes instead of default 60 seconds.
```

Comment:
The 100 / 1000 / 0.5 / 10 case is failing because of timeout. I ran it locally and it takes 258s (well above 2mins). So this test would have to be removed or the timeout should be increased to ~5 mins. However, maybe this can be ignored for now since the optimization would bring it down to ~90s.